src/transformers/pipelines/token_classification.py

```diff
@@ -264,8 +276,13 @@ def gather_pre_entities(
             # Filter special_tokens, they should only occur
             # at the sentence boundaries since we're not encoding pairs of
             # sentences so we don't have to keep track of those.
-            if special_tokens_mask[idx]:
-                continue
+            try:
+                if special_tokens_mask[idx]:
+                    continue
+            except Exception:
+                import ipdb
```

Comment:
same (I debug in the exact same way btw haha)

Reply:
try ... except + ipdb is epic

Reply:
Ahaha so useful in those very hard to detect error cases ;)